pipeline/schema.py

```diff
@@ -33,7 +33,8 @@
     'ConditionalDownload',
     'NonInteractiveStream',
     'OnDemandStream',
-    'PermanentDownload'
+    'PermanentDownload',
+    'iHeartCustom'
```

Comment:
The rest of this is alphabetical, this change should follow that

Reply:
I had put it at the end since it's different than the others but I guess alpha works too.